apps/src/lib/node/ledger/shell/testing/node.rs

```diff
@@ -404,6 +405,7 @@ impl<'a> Client for &'a MockNode {
         let tx_bytes: Vec<u8> = tx.into();
         self.submit_tx(tx_bytes);
         if !self.success() {
+            resp.code = tendermint::abci::Code::Err(1337); // TODO: submit_tx should return the correct error code + message
```

Comment:
@batconjurer I had to add this otherwise it would keep try to send the tx (cc @grarco )